pkg/agent/openflow/multicluster.go

```diff
@@ -25,6 +25,11 @@ import (
 // for cross-cluster traffic to distinguish from in-cluster traffic.
 var GlobalVirtualMACForMulticluster, _ = net.ParseMAC("aa:bb:cc:dd:ee:f0")
 
+// UnknownLabelIdentity represents an unknown label identity.
+// 24 bits in VNI are used for label identity. The max value is reserved for
+// UnknownLabelIdentity.
+const UnknownLabelIdentity = uint32(0xffffff)
```

Comment:
I wonder why it doesn't use 0x0 as the unknown label identity?

Reply:
Because the default/nil value of tun_id is 0x0, the security drop flow may unexpectedly drop packets that are not under stretchedNetworkPolicy scope.